README.md

```diff
@@ -173,6 +174,51 @@ w('my-widget', properties, children);
 The example above that uses a string for the `widgetConstructor `, is taking advantage of the [widget registry](#widget-registry) functionality.
 The widget registry allows for the lazy loading of widgets.
 
+### tsx
+
+In additional to the programatic functions `v` and `w`, widget-core supports the use of the `jsx` syntax known as [`tsx`](https://www.typescriptlang.org/docs/handbook/jsx.html) in TypeScript.
```

Comment:
To be slightly more explicit that this is an alternative approach:

widget-core supports the use -> widget-core optionally supports the use